portafly/src/components/AppNavExpandable.tsx

```diff
@@ -0,0 +1,24 @@
+import * as React from 'react'
+import {useRouteMatch} from 'react-router-dom'
+import {NavExpandable} from '@patternfly/react-core'
+import {AppNavItem, IAppNavItemProps} from './AppNavItem'
+
+export interface IAppNavExpandableProps {
+  title: string
+  to: string
+  items: Array<IAppNavItemProps | undefined>
+}
+
+export const AppNavExpandable: React.FunctionComponent<IAppNavExpandableProps> = ({title, to, items}) => {
+  const match = useRouteMatch({
+    path: to,
+  })
+  const isActive = !!match
+  return (
+    <NavExpandable title={title} isActive={isActive} isExpanded={isActive}>
+      {items.map((subNavItem, jdx) => (
+        <AppNavItem {...subNavItem} key={jdx}/>
```

Comment:
```suggestion
        <AppNavItem {...subNavItem} key={subNavItem.id}/>
```

Reply:
All changes regarding best practices will be address in a following PR ;)